tools/projmgr/src/ProjMgrWorker.cpp

```diff
@@ -479,7 +479,9 @@ bool ProjMgrWorker::ProcessDevice(ContextItem& context) {
       }
     }
     if (!matchedDevice) {
-      ProjMgrLogger::Error("specified device '" + deviceItem.name + "' was not found");
+      string msg = "specified device '" + deviceItem.name + "' was not found among the installed packs.";
+      msg += "\nuse \'cpackget\' utility to install software packs.\n  cpackget add Vendor.PackName --pack -root ./Path/Packs";
```

Comment:
The cpackget `--pack-root` option should be corrected here and in the test cases:
```suggestion
      msg += "\nuse \'cpackget\' utility to install software packs.\n  cpackget add Vendor.PackName --pack-root ./Path/Packs";
```